aws_lambda_function.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_lambda_function" "ami_encryption_lambda" {
+  count         = "${var.encrypt_ami}"
+  filename      = "${data.archive_file.ami_encryption.0.output_path}"
+  description   = "Responsible for creating AMI with encrypted root volume."
+  function_name = "${var.environment}-AMI-ENCRYPTION-FUNCTION"
```

Comment:
Consider putting a `upper()` function around the environment variable as all the rest of the string is in uppercase characters.

Reply:
added :)